lib/stateTransition/createStateTransitionFactory.js

```diff
@@ -52,9 +52,11 @@ function createStateTransitionFactory() {
         throw new InvalidStateTransitionTypeError(rawStateTransition);
     }
 
-    stateTransition
-      .setSignature(rawStateTransition.signature)
-      .setSignaturePublicKeyId(rawStateTransition.signaturePublicKeyId);
+    stateTransition.setSignature(rawStateTransition.signature);
```

Comment:
Maybe we should set signatures via constructors (abstract) as a part of a raw transition data?